microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -33,42 +95,89 @@ To submit suggestions and feedback, enter an [Issue about the Add-ons API](https
+      * The **Expiry date** of each Secret.
 
    ![The 'Publish API' page at Partner Center after clicking 'Create API credentials', now showing Client ID, Client Secret, and Auth Token URL](./using-addons-api-images/create-api-credentials-button.png)
 
-1. Write down the **Client ID**, **Client secret** and the **Access token URL**.  You'll use these values in the next step, to get an access token.
+1. Write down the following:
+   * The **Client ID**.
+   * The **Access token URL**.
+   * The new **Client secret**.
+
+   You'll use the Client ID, access token URL, and client secret to get an access token, per [Retrieve the access token](#retrieve-the-access-token), below .  You'll then use the access token in the request header when using a REST endpoint.
 
 > [!IMPORTANT]
-> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials). This particular secret isn't shown again.
+> Be sure to write down the client secret now, because it's only visible immediately after enabling or renewing the API (that is, after creating API credentials).  This particular secret isn't shown again.
 
-You can generate multiple client secrets for your client ID.  For example, you can create multiple secrets for multiple projects.
+You can generate multiple client secrets for your Client ID.  For example, you can create multiple secrets for multiple projects.
 
 
-<!-- ====================================================================== -->
-## Retrieving the access token
+<!-- ---------------------------------- -->
+#### Retrieve the access token
 
-After you've acquired the necessary authorization for your application, get access tokens for APIs.  To get a token using the client credentials grant, send a POST request to the Access token URL (the OAuth token).  The tenant information is available in the URL that you received in the **Before you begin** steps above.
+You obtain an access token by sending a POST message to an access token URL.  The Partner Center UI shows the access token URL, which is the same as in this article.  You then include the access token in the header when using an API endpoint, such as the endpoint for [Uploading a package to update an existing submission](#uploading-a-package-to-update-an-existing-submission), below.
+
+<!-- todo: old wording; reword/ condense/ delete: -->
+After you've acquired the necessary authorization for your application, get access tokens<!-- todo: plural or singular?  one token per endpoint? --> for APIs.  To get a token (an OAuth token) by using the client credentials grant,<!-- todo: define --> send a POST request to the Access token URL.  The tenant information<!-- todo: define, what relevance? which pieces of info? --> is available in the access token URL that you received in the previous step, [Enable the Update REST API at Partner Center](#enable-the-update-rest-api-at-partner-center).
```

Comment:
clarify abstruse paragraph: can we make this seem simple, instead of making it seem complicated?  
why get multiple access tokens?  
what's "using the client credentials grant"?  
what's "the tenant information" - which pieces of info, specifically?

@bhuvanapriyap 

Moved "(an OAuth token) to start of sentence.  
Added clear simple sentences above paragraph, at start of section.
Added clear lead-ins before next 3 code blocks.